GOVERNANCE.md

```diff
@@ -194,11 +193,19 @@ Voting for first election is restricted to establish stable governance, and to d
 ### Core Contributors
 Core Contributors are those who have provided consistent and meaningful contributions to ArviZ.
 These can be, but are not limited to, code contributions, community contributions, tutorial
-development etc.
+development etc. Core Contributors will be given the ability to manage the ArviZ github
+repository, including code merges to master. This does not necessarily mean Core Contributors
+must submit code, but more so signifies trust with the project as a whole.
+
+#### Core Contributor Responsibilities
+* Enforce code of conduct
+* Maintain a check against Council
 
 #### Core Contributor Nominations and Confirmation Process
-Current Core Contributors can nominate candidates for consideration by the council. The council
-can make the determination for acceptance with a process of their choosing.
+Current Core Contributors can nominate candidates for consideration by the council. On the first
+of the month following a nomination the Council will vote on each nominee where greater than 50% of the votes
+means a confirmation. In the event of a tie a second election will occur where all core contributors can vote,
+if 50% or greater of the community confirms the nominee will be accepted.
```

Comment:
I start here.

What options do we have?

Binary voting
--> majority rules: at least 50 % of the votes
--> minority rukes: at least x % of the votes (x < 50%)
--> ... rules: at least y % of the votes (y > 50%)
--> all agree: 100 % of the votes

What other systems are possible?

Reply:
I think we should go with yes-no-neutral as we were discussing to do with council elections. In that case acceptance could be decided on a having more yes than no base, or as having yes go over a threshold (without taking into account if the rest of votes are no or neutral), or having two thresholds (i.e. yes larger than X% and no below Y%), etc. @AlexAndorra probably knows better the weakness and strengths of each system

Reply:
As this is a binary choice here (one proposal at a time) and not a multichoice setting (several proposals compete against each other at the same time), we can't use Majority Judgment, but I like the yes-no-neutral and two-threshold process! 
That way, we can ensure that a majority of the community likes the proposed idea _and_ there isn't a substantial part of the community that dislikes the idea. Also, it's the way we elect Council Members in the first round, so it ties things up nicely 😉 
Setting the thresholds is hard and artificial, but what about yes >= 50% and no <= 25% (i.e at least half of community likes and not more than a fourth of it dislikes)?

Reply:
I am no opposed to idea but have question. Per NumFOCUS's second comment, and critical to this voting style, what happens if both no and yes thresholds are achieved? How do we resolve?

Reply:
Both conditions are necessary and sufficient to pass a proposal: yes >= 50% _and_ no <= 25%. If one of these conditions isn't met -- e.g yes = 60% but no = 30% -- the proposal is refused and must be amended before being put to the vote again. The same goes if no = 10% but yes = 40% of course.

Reply:
@AlexAndorra I don't think council election section says anything about yes-neutral-no voting nor about the 2 thresholds, it should probably be udated together with some rules to change the number of council members

Reply:
How do we amend a core contributor add though? That's a pretty binary proposal so if we hit both the yes and no threshold I don't get what the next step is

Reply:
@OriolAbril I think I answered you on Slack.
@canyon289 I'm not sure about what you mean by "amend a core contributor add", but, in your example, if the no threshold is hit, then the proposed core contributor is rejected, even though the yes threshold is hit too -- both conditions must be true (yes >= 60% _and_ no <= 20%) for the core contributor to be accepted.

Reply:
Core contributor add meant if we were looking to add a core contributor. Ok this makes sense now ( i was just too slow to get it)